### ISSUE TITLE
Ajouter la diffusion WFS dans les options du formulaire

### ISSUE THREAD

**Issue by flacombe** [openpcrs/pcrs.beta.gouv.fr#411]
Nous avons une demande d'évolution mineure pour ajouter l'option WFS dans la liste des possibilités de diffusion des livrables.
![image](https://github.com/openpcrs/pcrs.beta.gouv.fr/assets/5690599/11aab133-abbc-45c0-b97e-f5e6916a9fc2)

Cela demande une évolution du modèle de données (dans la doc du moins).

Du reste, puis-je rajouter l'option dans le formulaire via une simple PR ou il faut faire un truc dans la base de données @GllmR ?
Si c'est juste rajouter l'option html et une entrée dans une liste de validation, je peux le faire tout seul.

**Comment by GllmR** [comment]
> Nous avons une demande d'évolution mineure pour ajouter l'option WFS dans la liste des possibilités de diffusion des livrables. ![image](https://private-user-images.githubusercontent.com/5690599/323562431-11aab133-abbc-45c0-b97e-f5e6916a9fc2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1MTIzNTcsIm5iZiI6MTcxMzUxMjA1NywicGF0aCI6Ii81NjkwNTk5LzMyMzU2MjQzMS0xMWFhYjEzMy1hYmJjLTQ1YzAtYjk3ZS1mNWU2OTE2YTlmYzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTlUMDczNDE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njc0OTBiNWQ3Y2E1NTJjNDE5NzRmZWFiNjNiYjFhZGE2N2EzYjVhNmY1N2UwZmIxNWE5NDQwMGEwZDg0MjkxOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ze0_X3laxPMB5yYQWmxV_4MWz3xT9vN3eXbWu7Tjj8M)
> 
> Cela demande une évolution du modèle de données (dans la doc du moins).
> 
> Du reste, puis-je rajouter l'option dans le formulaire via une simple PR ou il faut faire un truc dans la base de données @GllmR ? Si c'est juste rajouter l'option html et une entrée dans une liste de validation, je peux le faire tout seul.

Oui, ça doit être à peu près ça, mais de mémoire, il faut aussi que tu mettes à jour les tests.

**Comment by flacombe** [comment]
Merci @GllmR pour la réponse, je vais m'en occuper.

On va aussi ajouter le DXF au format des livrables de la même façons
